flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/SchedulerBase.java

```diff
@@ -397,6 +407,18 @@ protected JobGraph getJobGraph() {
 
 	protected abstract long getNumberOfRestarts();
 
+	private Map<ExecutionVertexID, ExecutionVertexVersion> incrementVersionsOfAllVertices() {
+		return executionVertexVersioner.recordVertexModifications(
+			IterableUtils.toStream(schedulingTopology.getVertices())
+				.map(SchedulingExecutionVertex::getId)
+				.collect(Collectors.toSet()));
+	}
+
+	@VisibleForTesting
+	ExecutionVertexVersioner getExecutionVertexVersioner() {
+		return executionVertexVersioner;
+	}
```

Comment:
Can be removed, as we pass in the `executionVertexVersioner` and can keep a reference to it.